src/librawspeed/decoders/NefDecoder.cpp

```diff
@@ -76,7 +76,7 @@ RawImage NefDecoder::decodeRawInternal() {
   TiffEntry *offsets = raw->getEntry(STRIPOFFSETS);
   TiffEntry *counts = raw->getEntry(STRIPBYTECOUNTS);
 
-  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100 ") { /**Sigh**/
+  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100") { /**Sigh**/
```

Comment:
That is how I did interpret it as well ("D100 in general"). If Nikon should release another model with the same name, then it might or might not be supported with or without this change. I don't think this is relevant.